### ISSUE TITLE
Implement the WebSocket constructor that takes DOMString[]

### ISSUE THREAD

**Issue by jdm** [servo/servo#6696]
Or else `sequence<DOMString>` like Gecko, since `T[]` was removed from WebIDL. Depends on #544.

**Comment by boghison** [comment]
This depends on an open issue, that depends on a PR that was merged? Are there still any obstacles?

**Comment by jdm** [comment]
Nope!

**Comment by boghison** [comment]
@jdm Then #544 should be closed, shouldn't it?

**Comment by jdm** [comment]
No, #544 has not been resolved.

**Comment by boghison** [comment]
@jdm Sequence arguments are available, but not sequence return values. Is that correct?

**Comment by jdm** [comment]
No, we do not support either:
* http://mxr.mozilla.org/servo/source/components/script/dom/bindings/codegen/CodegenRust.py#1326
* http://mxr.mozilla.org/servo/source/components/script/dom/bindings/codegen/CodegenRust.py#751

**Comment by KiChjang** [comment]
Superseded by #9053.
